### PR TITLE
tarsum: add a big fat warning

### PR DESCRIPTION
**Pull request by vbatts**
@calavera @jfrazelle @icecrime @stevvooe @dmcgowan 

Just making this point clear to folks ...

Signed-off-by: Vincent Batts vbatts@redhat.com

### CODE REVIEW

**Comment by dmcgowan** [comment]
:smile_cat:
 
LGTM

**Comment by LK4D4** [comment]
ping @thaJeztah @moxiegirl

**Comment by thaJeztah** [comment]
It's big, fat, so LGTM

**Comment by stevvooe** [comment]
LGTM

"attestation" - word choice. :smiley_cat:

File: pkg/tarsum/tarsum_spec.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ This document describes the algorithms used in performing the TarSum checksum
 calculation on filesystem layers, the need for this method over existing
 methods, and the versioning of this calculation.
 
+## Warning
+
+This checksum algorithm is for best-effort comparision of file trees with fuzzy logic.
```
Comment (thaJeztah) [inline pkg/tarsum/tarsum_spec.md]: Oh! "comparision"? Sorry, didn't spot that when reading on my phone earlier :sweat_smile:

Comment (vbatts) [inline pkg/tarsum/tarsum_spec.md]: Fixed

**Comment by thaJeztah** [comment]
I don't think we need more LGTMs. Merging :innocent:

**Comment by moxiegirl** [comment]
@thaJeztah Wah, you diss'd my LGTM.  I give it in any case you :imp:  bahwahahahahahah

**Comment by thaJeztah** [comment]
@moxiegirl I gathered it wasn't really docs-docs, so :wink: :innocent:

**Comment by moxiegirl** [comment]
image

**Comment by moxiegirl** [comment]
bwahahahah. It was difficult to find a grumpy picture of the queen. It is good to be queen apparently.

**Pull request merged**